operators/pkg/instctrl/cloudinit_test.go

```diff
@@ -102,35 +103,36 @@ var _ = Describe("Generation of the cloud-init configuration", func() {
 		environment = clv1alpha2.Environment{Name: environmentName}
 		tenant = NewTenant("user", workspaceName, clv1alpha2.User, []string{"tenant-key-1", "tenant-key-2"})
 
-		webdavSecretName = types.NamespacedName{Namespace: instanceNamespace, Name: webdavCredentials}
+		pvcSecretName = types.NamespacedName{Namespace: instanceNamespace, Name: userPvcSecret}
 		objectName = forge.NamespacedName(&instance)
 		secret = corev1.Secret{}
 
 		ownerRef = metav1.OwnerReference{
 			APIVersion:         clv1alpha2.GroupVersion.String(),
-			Kind:               "Instance",
-			Name:               instance.GetName(),
-			UID:                instance.GetUID(),
+			Kind:               "Tenant",
+			Name:               tenant.GetName(),
+			UID:                tenant.GetUID(),
 			BlockOwnerDeletion: pointer.BoolPtr(true),
 			Controller:         pointer.BoolPtr(true),
 		}
 	})
 
-	ForgeWebDavSecret := func(usernameKey, passwordKey string) *corev1.Secret {
+	ForgePvcSecret := func(serviceIpKey, serviceNameKey, servicePathKey string) *corev1.Secret {
 		return &corev1.Secret{
-			ObjectMeta: forge.NamespacedNameToObjectMeta(webdavSecretName),
+			ObjectMeta: forge.NamespacedNameToObjectMeta(pvcSecretName),
 			Data: map[string][]byte{
-				usernameKey: []byte(webdavUsername),
-				passwordKey: []byte(webdavPassword),
+				serviceIpKey:   []byte(nfsServiceIp),
+				serviceNameKey: []byte(nfsServiceName),
+				servicePathKey: []byte(nfsServicePath),
 			},
 		}
 	}
 
 	JustBeforeEach(func() {
 		client := FakeClientWrapped{Client: clientBuilder.Build()}
 		reconciler = instctrl.InstanceReconciler{
-			Client: client, Scheme: scheme.Scheme, WebdavSecretName: webdavCredentials,
-			ServiceUrls: instctrl.ServiceUrls{NextcloudBaseURL: nextcloudBaseURL},
+			Client: client, Scheme: scheme.Scheme,
+			ServiceUrls: instctrl.ServiceUrls{},
```

Comment:
This could be omitted, when structs are nested without being pointers, they're implicitly set as empty already.

Reply:
Ok, removed useless items